### DISCUSSION TITLE
Two-factor authentication pc desktop sms my country out

### DISCUSSION THREAD

**Discussion by seldefense** [community/community#24019] [New to GitHub]
<p>Your system sent me an email asking me to add a security authentication.<br>
I have a computer (PC Windows 10 64bits) where I do the programming.</p>
<ul>
<li>I don’t have a smartphone.</li>
<li>I have an old cell phone without internet that receives sms, this cell phone does not use applications because it is old. I can receive SMS but my country PERU (Lima) is not on the list of countries.</li>
</ul>
<p>What I can do?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You could use a TOTP authentication application on your computer. It’s a little weaker in terms of security because it’s still on the same device, but it should work.</p>
<p>Mozilla has a list of desktop and mobile options in step one of their 2FA instructions:<br>
<a href="https://support.mozilla.org/en-US/kb/secure-firefox-account-two-step-authentication" class="onebox" target="_blank" rel="noopener">https://support.mozilla.org/en-US/kb/secure-firefox-account-two-step-authentication</a></p>
<p>I personally like KeepassXC (at the bottom of the list), but haven’t tried its TOTP feature yet.</p>

**Comment by seldefense** [comment]
<p>Thank you very much, I just managed to do it.</p>
